WebHostLib/customserver.py

```diff
@@ -335,7 +335,7 @@ async def start_room(room_id):
                     with (db_session):
                         # ensure the Room does not spin up again on its own, minute of safety buffer
                         room = Room.get(id=room_id)
-                        room.last_activity = datetime.datetime.utcnow() - \
+                        room.last_activity = Utils.utcnow() - \
                                              datetime.timedelta(minutes=1, seconds=room.timeout)
```

Comment:
might as well join these lines now that we lowered the character count
```suggestion
                        room.last_activity = Utils.utcnow() - datetime.timedelta(minutes=1, seconds=room.timeout)
```